samples/ar-view-node/src/main/res/layout/view_node_text_view_annotation.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="100dp"
+    android:layout_height="100dp"
+    android:background="@drawable/rectangle_rounded_corner"
+    android:gravity="center">
+
+    <TextView
```

Comment:
Can the `TextView` be a root `View`?

Reply:
This is possible.
I removed the LinearLayout and added "android:background="@drawable/rectangle_rounded_corner" to the TextView.